.agents/skills/wpilib-sim/scripts/find_latest_simlog.py

```
@@ -0,0 +1,17 @@
+"""Compatibility wrapper for the wpilog-decode skill."""
+
+from __future__ import annotations
+
+import subprocess
+import sys
+from pathlib import Path
+
+
+def main() -> None:
+    target = Path(__file__).resolve().parents[2] / "wpilog-decode/scripts/find_latest_simlog.py"
+    subprocess.run([sys.executable, str(target), *sys.argv[1:]], check=True)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)